main.go

```diff
@@ -512,7 +500,14 @@ func main() {
 			return err
 		}
 
-		api, closer, err := lcli.GetGatewayAPI(cctx)
+		// send a clean cctx down into lotus - it needs only the node "api-url"
```

Comment:
```suggestion
		// send a CLI context to lotus that contains only the node "api-url" flag set, so that other flags don't accidentally conflict with lotus cli flags
```